Ev2Specs/ServiceGroupRoot/Shell/Run/Run.ps1

```diff
@@ -0,0 +1,77 @@
+# ensure SAS variables were passed in
+if ($env:SECRET_MANAGEMENT_MODULE -eq $null)
+{
+    Write-Verbose -Verbose "SECRET_MANAGEMENT_MODULE variable didn't get passed correctly"
+    return 1
+}
+
+if ($env:SECRET_MANAGEMENT_VERSION -eq $null)
+{
+    Write-Verbose -Verbose "SECRET_MANAGEMENT_VERSION variable didn't get passed correctly"
+    return 1
+}
+
+if ($env:DESTINATION_ACR_NAME -eq $null)
+{
+    Write-Verbose -Verbose "DESTINATION_ACR_NAME variable didn't get passed correctly"
+    return 1
+}
+
+if ($env:DESTINATION_ACR_URI -eq $null)
+{
+    Write-Verbose -Verbose "DESTINATION_ACR_URI variable didn't get passed correctly"
+    return 1
+}
+
+if ($env:MI_CLIENTID -eq $null)
+{
+    Write-Verbose -Verbose "MI_CLIENTID variable didn't get passed correctly"
+    return 1
+}
+
+
+try {
+    Write-Verbose -Verbose "SecretManagement: $env:SECRET_MANAGEMENT_MODULE"
+    Write-Verbose -Verbose "Version: $env:SECRET_MANAGEMENT_VERSION"
+    Write-Verbose -Verbose "acrname: $env:DESTINATION_ACR_NAME"
+    Write-Verbose -Verbose "acruri: $env:DESTINATION_ACR_URI"
+    Write-Verbose -Verbose "MI client Id: $env:MI_CLIENTID"
+
+    $secretManagementFileName = "Microsoft.PowerShell.SecretManagement.$($env:SECRET_MANAGEMENT_VERSION).nupkg"
+
+    Write-Verbose -Verbose "Download files"
+    Invoke-WebRequest -Uri $env:SECRET_MANAGEMENT_MODULE -OutFile $secretManagementFileName
```

Comment:
We're both just curious what the original value of `SECRET_MANAGEMENT_MODULE` is, and why we have to "download" it in the container. Like, is it a local URI?

Reply:
The module gets uploaded to a storage blob by Ev2, and we need to download from there.